Source/Scene/ClippingPlanesCollection.js

```diff
@@ -0,0 +1,189 @@
+         * @default 0.0
+         */
+        this.edgeWidth = defaultValue(options.edgeWidth, 0.0);
+    }
+
+    var scratchPlane = new Plane(Cartesian3.UNIT_X, 0.0);
+    var scratchMatrix = new Matrix4();
+    /**
+     * Applies the transformations to each plane and packs it into an array.
+     *
+     * @param viewMatrix
+     * @param [array]
+     * @returns {Cartesian4[]} The array of packed planes.
+     */
+    ClippingPlanesCollection.prototype.transformAndPackPlanes = function (viewMatrix, array) {
+        var planes = this.planes;
+        var length = planes.length;
+
+        if (!defined(array)) {
+            array = new Array(length);
+        }
+
+        var transform = Matrix4.multiply(viewMatrix, this.transformationMatrix, scratchMatrix);
+
+        for (var i = 0; i < length; ++i) {
+            var plane = planes[i];
+            var packedPlane = array[i];
+
+            Plane.transform(plane, transform, scratchPlane);
+
+            Cartesian3.clone(scratchPlane.normal, packedPlane);
+            packedPlane.w = scratchPlane.distance;
+        }
+
+        return array;
+    };
+
+    /**
+     * Duplicates this ClippingPlanesCollection instance.
+     *
+     * @param {ClippingPlanesCollection} [result] The object onto which to store the result.
+     * @returns he modified result parameter or a new ClippingPlanesCollection instance if one was not provided.
+     */
+    ClippingPlanesCollection.prototype.clone = function (result) {
+        if (!defined(result)) {
+            result = new ClippingPlanesCollection();
+        }
+
+        result.planes = Array.from(this.planes);
+        result.enabled = this.enabled;
+        Matrix4.clone(this.transformationMatrix, result.transformationMatrix);
+        result.inclusive = this.inclusive;
+        Color.clone(this.edgeColor, result.edgeColor);
+        result.edgeWidth = this.edgeWidth;
+
+        return result;
+    };
+
+    /**
+     * Determines the type intersection with the planes of this bounding collection and the specified {@link BoundingVolume}.
+     *
+     * @param {BoundingVolume} boundingVolume The volume to determine the intersection with the planes.
+     * @param {Matrix4} [parentTransform] An optional, additional matrix to transform the plane to world coordinates.
+     * @returns {Intersect} {@link Intersect.INSIDE} if the entire volume is on the side of the planes
+     *                      the normal is pointing and should be entirely rendered, {@link Intersect.OUTSIDE}
+     *                      if the entire volume is on the opposite side and should be clipped, and
+     *                      {@link Intersect.INTERSECTING} if the volume intersects the planes.
+     */
+    ClippingPlanesCollection.prototype.computeIntersectionWithBoundingVolume = function (boundingVolume, parentTransform) {
+        var planes = this.planes;
+        var length = planes.length;
+
+        var transformation = this.transformationMatrix;
```

Comment:
Throughout, rename `transformation` to `transform` - it is more concise and better aligned with graphics terms.